lib/puppet/parser/functions/ht_crypt.rb

```diff
@@ -0,0 +1,17 @@
+module Puppet::Parser::Functions
+    newfunction(:ht_crypt, :type => :rvalue, :doc => <<-EOS
```

Comment:
I understand this function has been included as historically instance.pp will break without.

This function is a copy of the function provided by the `htpasswd` module. Best to have a dependency of the `htpasswd` module

Reply:
I've removed the file `lib/puppet/parser/functions/ht_crypt.rb` and added a dependency. However this dependency is ignored by `r10k` and therefore needs to be manually added to the Puppetfile of the project in question.